python/postgresql/postgresql/README.md

```diff
@@ -25,37 +25,47 @@ limitations under the License.
 
 1. [Install uv](https://docs.astral.sh/uv/getting-started/installation/)
 
-1. [Install dbc](https://docs.columnar.tech/dbc/getting_started/installation/)
-
-1. [Install PostgreSQL](https://www.postgresql.org/download/)
-   - On macOS, if you have Homebrew installed, run `brew install postgresql@17`
+2. [Install dbc](https://docs.columnar.tech/dbc/getting_started/installation/)
 
 ### Set up PostgreSQL
 
-1. Start PostgreSQL
-   - If you installed it with Homebrew, run `brew services start postgresql@17`
-1. Create a table in PostgreSQL and load data into it by running `psql -d postgres -f games.sql`
+1. [Install Docker](https://docs.docker.com/get-started/get-docker/)
+
+2. Start a PostgreSQL instance:
+
+    ```sh
+    docker run -d --rm --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 postgres
+    ```
+
+3. Create a table in PostgreSQL and load data into it:
+
+    ```sh
+    cat games.sql | docker exec -i some-postgres psql -d postgres -U postgres -W mysecretpassword
```

Comment:
`-W mysecretpassword` is not having the intended effect here. It gives me the warning:

```
psql: warning: extra command-line argument "mysecretpassword" ignored
```